src/components/common/GitInfoMaterial.tsx

```diff
@@ -209,6 +222,10 @@ export default function GitInfoMaterial({
                             noSearchResults={material.noSearchResult}
                             noSearchResultsMsg={material.noSearchResultsMsg}
                             clearSearch={clearSearch}
+                            appId={appId}
+                            handleGoToWorkFlowEditor={() => {
+                                goToWorkFlowEditor()
```

Comment:
pass `goToWorkFlowEditor` directly as handleGoToWorkFlowEditor arg will be optional as discussed

Reply:
done